src/main/java/wolox/training/models/Book.java

```diff
@@ -19,6 +18,42 @@
     public Book() {
     }
 
+    public Book(Long id, String genre, @NotNull String author,
+            @NotNull String image, @NotNull String title,
+            @NotNull String subtitle, @NotNull String publisher,
+            @NotNull String year, @NotNull Integer pages,
+            @NotNull String isbn) {
+
+        this.id = id;
+        this.genre = genre;
+        this.author = author;
+        this.image = image;
+        this.title = title;
+        this.subtitle = subtitle;
+        this.publisher = publisher;
+        this.year = year;
+        this.pages = pages;
+        this.isbn = isbn;
+    }
+
+    public Book(Long id, String genre, @NotNull String author,
+            @NotNull String image, @NotNull String title,
+            @NotNull String subtitle, @NotNull String publisher,
+            @NotNull String year, @NotNull Integer pages,
+            @NotNull String isbn, List<User> users) {
+        this.id = id;
+        this.genre = genre;
+        this.author = author;
+        this.image = image;
+        this.title = title;
+        this.subtitle = subtitle;
+        this.publisher = publisher;
+        this.year = year;
+        this.pages = pages;
+        this.isbn = isbn;
+        this.users = users;
+    }
```

Comment:
don't need two constructors, remove one and send `users = null` when you don't need this